app/controllers/api/proxy.php

```diff
@@ -49,6 +49,12 @@
         if ($domain === $mainDomain) {
             throw new Exception(Exception::GENERAL_ARGUMENT_INVALID, 'You cannot assign your main domain to specific resource. Please use subdomain or a different domain.');
         }
+
+        $functionsDomain = App::getEnv('_APP_DOMAIN_FUNCTIONS', '');
```

Comment:
Should we do the same for regular custom domains? or this is taking care of this and the error message can be more accurate?

Reply:
I don't understand what you mean?